packages/ember-runtime/lib/system/native_array.js

```diff
@@ -54,10 +53,6 @@ let NativeArray = Mixin.create(MutableArray, Observable, Copyable, {
 
   // primitive for array support.
   replace(idx, amt, objects) {
-    if (this.isFrozen) {
```

Comment:
I don't think we can remove this (even though `Ember.Freezable` is deprecated, it is still fully supported until 3.0.0).

Can you swap this to a debug mode assertion though?

Reply:
will do

Reply:
@bekzod - Friendly ping...

Reply:
done